src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/GenotypingEngine.java

```diff
@@ -376,7 +376,7 @@ protected final boolean hasTooManyAlternativeAlleles(final VariantContext vc) {
     }
 
     protected boolean emitAllSites() {
```

Comment:
Put a comment here explaining this in somewhat better detail I think.

Reply:
I put a similar comment there and referred to the OutputMode documentation. Do you want more detail?
I also renamed this function to `emitAllActiveSites()`, just to be consistent.